### PR TITLE
ansible-logstash-callback

### PR DESCRIPTION
**Pull request by ujenmr**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

logstash-callback

##### ANSIBLE VERSION

```
ansible 2.1.2.0
```

##### SUMMARY

```
Logstash callback plugin, allows push ansible's stdout to logstash
```

### CODE REVIEW

File: lib/ansible/plugins/callback/logstash.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# (C) 2016, Ievgen Khmelenko <ujenmr@gmail.com>
+
```
Comment (sivel) [inline lib/ansible/plugins/callback/logstash.py:17]: This is missing a GPL v3 license preamble

**Comment by killabayte** [comment]
Great and really need thing, thank you very much!..

File: lib/ansible/plugins/callback/logstash.py (hunk 1 above)
Comment (ujenmr) [reply lib/ansible/plugins/callback/logstash.py:17]: Added https://github.com/ansible/ansible/pull/18282/commits/64d539549436501f375590dc615e09385f4bdb57

**Comment by missnebun** [comment]
+1

**Pull request merged**